thread/exports.cmake

```diff
@@ -33,6 +33,10 @@ omr_add_exports(j9thr_obj
 	omrthread_abort
 	omrthread_attach
 	omrthread_attach_ex
+	omrthread_waiting_to_acquire
+	omrthread_monitor_is_acquired
+	omrthread_monitor_getNumOfTimesAcquired
+	omrthread_monitor_getCurrentOwner
```

Comment:
Not an expert in this area but just eyeballing it seems all APIs in the thread library are snake_case where as this is some mix of snake_camelCase. Should we continue to be consistent and name these:

```
	omrthread_monitor_get_num_times_acquired
	omrthread_monitor_get_current_owner
```

Reply:
good point. I will modify the names.